src/ensembl/src/shared/components/instant-download/instant-download-fetch/fetchSequenceChecksums.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { gql } from '@apollo/client';
+
+import { client } from 'src/gql-client';
+
+type GeneFragment = {
+  transcript: {
+    product_generating_contexts: ProductGeneratingContextFragment[];
```

Comment:
Should we consider placing the definition of `ProductGeneratingContextFragment` above ?

Reply:
✅